app/views/layouts/application.html.erb

```diff
@@ -4,26 +4,39 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
+    <% flash.each do |name, msg| %>
+      <%= content_tag(:div, msg, class: "alert alert-#{name}") %>
+    <% end %>
+
     <div class="top-bar">
       <div class="top-bar-left">
         <ul class="dropdown menu" data-dropdown-menu>
           <li class="menu-text">Site Title</li>
+          <li><a href=<%= new_api_path %>>Add New</a></li>
+          <li><a href=<%= apis_path %>>View All by Category</a></li>
```

Comment:
Use the `link_to` helper instead of the full `<a>` tag, please! Docs are [here](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html).